### DISCUSSION TITLE
Where i can change I2C_speed in firmware ?

### DISCUSSION THREAD

**Discussion by skimans** [iNavFlight/inav#7702] [General]
I need to change the i2C_speed of default (400 khz) in 800 khz.
Where i can find this in the firmware ?
thanks

**Comment by OptimusTi** [comment]
![image](https://user-images.githubusercontent.com/45466510/147787317-28177854-f10f-435c-b5ba-a416a253bbbb.png)

**Reply by skimans** [reply]
mmm...i want change directly in the firmware and after compile it.

**Reply by CZEMacLeod** [reply]
Did you ever get a way of doing this?
